test/ScanDependencies/cannot_import_with_map_duplicate_entries.swift

```diff
@@ -0,0 +1,117 @@
+// UNSUPPORTED: OS=windows-msvc
+// RUN: %empty-directory(%t)
+// RUN: mkdir -p %t/clang-module-cache
+// RUN: mkdir -p %t/inputs
+// RUN: echo "public func foo() {}" >> %t/foo.swift
+// RUN: %target-swift-frontend -emit-module -emit-module-path %t/inputs/Foo.swiftmodule -emit-module-doc-path %t/inputs/Foo.swiftdoc -emit-module-source-info -emit-module-source-info-path %t/inputs/Foo.swiftsourceinfo -module-cache-path %t.module-cache %t/foo.swift -module-name Foo
+// RUN: %target-swift-emit-pcm -module-name SwiftShims %swift-lib-dir/swift/shims/module.modulemap -o %t/inputs/SwiftShims.pcm
+// RUN: %target-swift-emit-pcm -module-name _SwiftConcurrencyShims %swift-lib-dir/swift/shims/module.modulemap -o %t/inputs/_SwiftConcurrencyShims.pcm
+
+// RUN: echo "[{" > %/t/inputs/map_with_duped_swift_module.json
```

Comment:
Maybe it would be easier to read using `split-file`

```
// RUN: split-file %s %t
// RUN: sed -e "s|TEMP_DIRECTORY|%/t|g" %t/inputs/map_with_duped_swift_module.json.template > %t/inputs/map_with_duped_swift_module.json
// ...

//---foo.swift
public func foo() {}

//---inputs/map_with_duped_swift_module.json.template
[{
  "moduleName": "Foo",
  "modulePath": "TEMP_DIRECTORY/inputs/Foo.swiftmodule",
  "docPath": "TEMP_DIRECTORY/inputs/Foo.swiftdoc",
  "sourceInfoPath": "TEMP_DIRECTORY/inputs/Foo.swiftsourceinfo",
  "isFramework": false
},
...
```